src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -241,14 +241,24 @@ private static boolean shouldBeAloneOnLineWithAloneOption(Details details,
      * @param details details for validation.
      * @param targetSrcLine A string with contents of rcurly's line
      * @return true, if right curly should be alone on line
-     *         when ALONE_OR_SINGLELINE or SAME option is used.
+     *         when ALONE_OR_SINGLELINE or SAME
+     *         option is used.
      */
     private static boolean shouldBeAloneOnLineWithNotAloneOption(Details details,
                                                                  String targetSrcLine) {
         return shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
                 && !isSingleLineBlock(details);
     }
 
+    /**
+     * Checks weather right curly is followed by a semicolon.
```

Comment:
`Checks whether the right curly is followed by a semicolon.`

Reply:
done.